web_src/js/features/repo-issue-pr-form.js

```diff
@@ -1,12 +1,12 @@
-import Vue from 'vue';
+import {createApp} from 'vue';
 import PullRequestMergeForm from '../components/PullRequestMergeForm.vue';
 
 export default function initPullRequestMergeForm() {
   const el = document.getElementById('pull-request-merge-form');
   if (!el) return;
 
-  const View = Vue.extend({
+  const View = createApp({
```

Comment:
I think, this would work. I couldn't figure out how to get the respective template to render the relevant element.

`templates/repo/issue/view_content/pull.tmpl`